config/custom-environment-variables.coffee

```diff
@@ -0,0 +1,38 @@
+module.exports =
+  application:
+    scroll:
+      # https://github.com/lorenwest/node-config/issues/272#issuecomment-223146123
+      __name: "APPLICATION_SCROLL"
+      __format: "json"
+    singleDatabase:
+      __name: "APPLICATION_SINGLE_DATABASE"
+      __format: "json"
+    sounds:
+      muteAll:
+        __name: "APPLICATION_SOUNDS_MUTEALL"
+        __format: "json"
+
+  server:
+    port:        "SERVER_PORT"
+
+  services:
+    projectController:
+      endpoint:  "SERVICES_PROJECT_ENDPOINT"
+    projectDatabase:
+      endpoint:  "SERVICES_DATABASE_ENDPOINT"
+    systemDatabase:
+      endpoint:  "SERVICES_SYSTEM_DATABASE_ENDPOINT"
+    flock:
+      endpoint:  "SERVICES_FLOCK_ENDPOINT"
```

Comment:
By the looks of it these are based on the weaver-server custom env variables. Please delete everything besides the server.port and logging.* items.